src/fiatExchanges/PaymentMethodSection.tsx

```diff
@@ -117,14 +151,7 @@ export function PaymentMethodSection({
         </Text>
       </View>
 
-      <View style={styles.imageContainer}>
-        <Image
-          testID={`image-${sectionQuotes[0].getProviderName()}`}
-          source={{ uri: sectionQuotes[0].getProviderLogo() }}
-          style={styles.providerImage}
-          resizeMode="center"
-        />
-      </View>
+      {renderProviderInfo(quote)}
```

Comment:
Looks like you use `quote` here rather than `sectionQuotes[0]` which was previously being used... `quote` seems correct to use anyways since it's the argument to this function (though this function is only ever called with `sectionQuotes[0]` as an argument), but it makes me wonder why we're still using `sectionQuotes[0]` on line 147. Unrelated to this PR I guess, but should we actually be using `quote` there?

Reply:
quote should always be `sectionQuote[0]` here.. removed the arg and added a comment.